src/uct/api/uct.h

```diff
@@ -1326,6 +1347,35 @@ struct uct_ep_params {
 };
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Parameters for connecting a UCT endpoint by @ref
+ * uct_ep_connect_to_ep_v2.
+ */
+struct uct_ep_connect_to_ep_params {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_ep_connect_params_field. Fields not specified by this mask
+     * will be ignored.
+     */
+    uint64_t                            field_mask;
+
+    /**
+     * Remote device address.
+     */
+    const uct_device_addr_t             *device_addr;
+
+    /**
+     * Remote endpoint address.
+     */
+    const uct_ep_addr_t                 *ep_addr;
+
+    /**
+     * Remote iface address.
```

Comment:
maybe "interface" in comment, to be clearer?

Reply:
done